ner_v2/tests/postman/run_v2_tests.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import absolute_import
+import json
+import traceback
+import os
+
+
+def get_number_of_records_per_entity():
+    with open('data/number_range.json', 'r') as file:
+        data = json.load(file)
+    return len(data)
+
+
+def check_if_data_valid(entities):
+    # Doing basic sanity checking here.
+    # Check if all data files are valid json and there are same number of records in all of them
+    # Also check that every test case has the keys: input and expected
+    try:
+       number_of_records = get_number_of_records_per_entity()
+       for entity in entities:
+           file_name = 'data/' + entity + '.json'
+           with open(file_name, 'r') as file:
+               data = json.load(file)
+           if (len(data) != number_of_records):
+               print('Incorrect number of records in ' + file_name)
+               return False
+           for item in data:
+               if not all (k in item for k in ('input', 'expected')):
+                   print('Invalid json data in ' + file_name)
```

Comment:
E111 indentation is not a multiple of four
E1601 [1;31mprint statement used[0m